client/my-sites/site-settings/form-general.jsx

```diff
@@ -346,6 +347,114 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	visibilityOptionsComingSoon() {
+		const { fields, isRequestingSettings, eventTracker, siteIsJetpack, translate } = this.props;
+
+		const blogPublic = parseInt( fields.blog_public, 10 );
+		const wpcomComingSoon = parseInt( fields.wpcom_coming_soon, 10 );
+
+		return (
+			<FormFieldset>
+				{ ! siteIsJetpack && (
+					<>
+						<FormLabel className="site-settings__visibility-label">
+							<FormRadio
+								name="blog_public"
+								value="-1"
+								checked={ -1 === blogPublic && 1 === wpcomComingSoon }
+								onChange={ () =>
+									this.handleVisibilityOptionChange( {
+										blog_public: -1,
+										wpcom_coming_soon: 1,
+									} )
+								}
+								disabled={ isRequestingSettings }
+								onClick={ eventTracker( 'Clicked Site Visibility Radio Button' ) }
+							/>
+							<span>{ translate( 'Coming soon' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **62** times:
`translate( 'Coming Soon' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/coming-soon-privacy-picker)